keyboards/geonworks/frogmini/readme.md

```diff
@@ -0,0 +1,49 @@
+# GeonWorks Frog Mini PCB firmware
+
+![FM_image](https://i.imgur.com/mhCsc6M.jpg)
```

Comment:
This uses smaller, but still high res version of the image that is more suitable for github markdown rendering.
```suggestion
![FM_image](https://i.imgur.com/mhCsc6Mh.jpg)
```

Reply:
Committed in [c8f5183](https://github.com/qmk/qmk_firmware/pull/17765/commits/c8f518338dcbf4444355bd9fa3f72710a05d20ef)